docs/tutorials/add-storage-adapter.md

```diff
@@ -34,18 +34,18 @@ If everything goes well, create a new pull request in [utopia-php/storage](https
 ### Phase 2
 In this phase we will add support to the new storage adapter in Appwrite.
 
-* Note for this to happen, your PR in the first phase should have been merged and new version of [utopia-php/storage](https://github.com/utopia-php/storage) library released.
+* Note for this to happen, your PR in the first phase should have been merged and a new version of [utopia-php/storage](https://github.com/utopia-php/storage) library released.
 
 ### Upgrade the utopia-php/storage dependency
 Upgrade the utopia-php/storage dependency in `composer.json` file.
 
 ### Introduce new environment variables
-If required for the new adapter, may be for credentials, introduce new environment variables. The storage environment variables are prefixed as `_APP_STORAGE_DEVICE`. Please read [Adding Environment Variables]() guidelines in order to properly introduce new environment variables.
+If required for the new adapter, maybe for credentials, introduce new environment variables. The storage environment variables are prefixed as `_APP_STORAGE_DEVICE`. Please read [Adding Environment Variables]() guidelines in order to properly introduce new environment variables.
```

Comment:
Suggested phrasing:
```
Introduce new environment variables if the adaptor requires new configuration information or to pass in credentials. The storage environment variables are prefixed as `_APP_STORAGE_DEVICE`. Please read [Adding Environment Variables]() guidelines in order to properly introduce new environment variables.
```

Notice how `[Adding Environment Variables]()` is also broken? This isn't your fault, but it would be great if you can fix this, too.

Reply:
is it correct now??

Reply:
I have updated it